platform-configuration/src/main/java/org/codice/ddf/configuration/ConfigurationManager.java

```diff
@@ -115,12 +112,14 @@
      * The organization that this instance of DDF is running for
      */
     public static final String ORGANIZATION = "organization";
-    
+
     /**
      * Site (email) contact
      */
     public static final String CONTACT = "contact";
 
+    private static final Logger logger = LoggerFactory.getLogger(ConfigurationManager.class);
```

Comment:
What's the convention for static final fields and constants? Do they have to be all caps?